### DISCUSSION TITLE
Thanks to github community

### DISCUSSION THREAD

**Discussion by mahbubsk** [community/community#28093] [Other Feature Feedback, Questions, & Ideas]
-  I wondered when I saw the "Badge" panel on my Github profile. I really love the feature.
- A lot of features continuously surprised me. Although I'm new here I'm really enjoying GitHub <3 
```
Take love <3
```

**Comment by jfullstackdev** [comment]
you can also introduce yourself at the official thread here

https://github.com/orgs/community/discussions/23204
